.cbmc-batch/jobs/aws_cryptosdk_md_abort/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+# if Makefile.local exists, use it. This provides a way to override the defaults 
+sinclude ../Makefile.local
+#otherwise, use the default values
+include ../Makefile.local_default
+
+UNWINDSET += 
+
+# Adds check for memory leaks for this proof
+CBMCFLAGS += --memory-leak-check
```

Comment:
Why this is applicable for this proof only, but not all the others? If all the proofs should also have this check, add on `Makefile.common`; otherwise, explain it in the comment.

Reply:
Done